arrays.js

```diff
@@ -145,14 +180,21 @@ Task 7:
 **************************************************************/
 function updateGrades(curve, students) {
   //TODO: Add your code here
+  return students.map(function (student) {
```

Comment:
Bravo! However, this approach will only output the updated grades, whereas what I wanted is to see the entire objects with the updated grades. Can you think of other ways to achieve that output? I'll be nice and give you a hint: "try using the spread operator" 🫡